### DISCUSSION TITLE
Push notifications not working in CodeMagic deployment

### DISCUSSION THREAD

**Discussion by ghost** [codemagic-ci-cd/codemagic-docs#2453] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

When I run the codemagic build for my Ios app, everything works fine except for the push notifications. We get the following error from apple:

ITMS-90078: Missing Push Notification Entitlement - 

Is there a line of code I can add to the codemagic.yaml that automatically adds the capability to the app?

### Expected results

Push notifications are received by app users

### Actual results

Error message from apple saying missing push notification entitlement

### Build id (optional)
660d57ee843e57c2896d027f

**Comment by ghost** [comment]
A similar question was asked here: https://github.com/orgs/codemagic-ci-cd/discussions/2338

**Reply by himesh-cm** [reply]
Hi @OlafNekemanAmberflow, I am investigating the issue and will keep you posted.

**Reply by himesh-cm** [reply]
Hello @OlafNekemanAmberflow, 

Once you have confirmed that you have enabled APNS for your profile, please save the profile again in your app store developer account. After that, re-upload it under `Teams > Settings > Codesigning Identities` and re-run the build. 

Please let me know once you have completed these steps and inform me if you still receive emails from Apple.

**Reply by himesh-cm** [reply]
@OlafNekemanAmberflow, Please note that you should contact our team through the chat widget in the UI for a quicker response, as your team is on a paid plan.

**Reply by ghost** [reply]
Thanks for the reply. However this is not the fix. In the mean time we have switched to another method, so you can close this ticket

**Reply by himesh-cm** [reply]
Sure, While I’m closing our discussion for now, if you have any other questions or suggestions, feel free to reply back or start a new discussion anytime. Have a nice day!

**Comment by SpeedotrackGPS** [comment]
Hello, I am also facing the same issue in Android push notification is working but with iOS Push Notification is not working while I am deploying from Flutterflow. Please assist me how to solve this issue
